apps/src/templates/progress/ProgressLesson.jsx

```diff
@@ -109,7 +107,5 @@ const ProgressLesson = React.createClass({
 export const UnconnectedProgressLesson = ProgressLesson;
 
 export default connect(state => ({
-  viewAs: state.stageLock.viewAs,
-  sectionId: state.sections.selectedSectionId,
-  hiddenStageState: state.hiddenStage,
+  lessonIsVisible: (lesson, viewAs) => lessonIsVisible(lesson, state, viewAs)
```

Comment:
Possibly contentious creating methods in our connect? Thought we already do this elsewhere when we map dispatch to props.